index.md

```diff
@@ -1,6 +1,61 @@
 ---
-# You don't need to edit this file, it's empty on purpose.
-# Edit theme's home layout instead if you wanna make some changes
-# See: https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-layout: home
+layout:     page
+title:      The Literature Review site of VITAL
+hidden:     true  # Excluded from the menu
 ---
+
+<style>
+    li {
+        margin-bottom: 0.7rem; /* This will make the lists easier to read. */
+    }
+</style>
+
+**Welcome!** Through this website, we will help each other stay up to date with new advances in our ever changing field. Our paper summaries will also help new members of VITALab to acquire essential knowledge about the state of the art. On this site, you'll find all the essential papers the VITALab members should be aware of.
+
+## How to add a review
+
+The process for adding reviews is _git-centric_. Basically, **you just need to add a file to the repo and make a pull request**. Let's go into the details :
+
+1.  Clone [the `vitalab.github.io` repo](https://github.com/vitalab/vitalab.github.io) on your computer.
+2.  Determine the category in which you will add your post. Categories are managed using folders :  
+    
+    ~~~
+    _posts/           # A post added here will have no category
+    course/
+        _posts/       # A post added here will be in the "course" category
+    machine-learning/
+        _posts/       # Same thing for the "machine-learning" category
+    ...               # There are other categories, you can add one too.
+    ~~~
+3.  Create the file `YYYY-MM-DD-title-of-your-review.markdown` in right folder (see above).  
```

Comment:
Create a file 'YYYY-MM-DD-title-of-your-review.markdown` and put it in the right folder (see above).